// Copyright 2013 Alberto Rojas (rojas.alberto@gmail.com)
// All rights reserved.

syntax = "proto2";

import "rh.proto";

package larxyz.rh;

message RelayConfig {
  optional Relay relay = 1;
  optional int32 hardware_id = 2;
}

message SensorConfig {
  optional Sensor sensor = 1;
  optional int32 hardware_id = 3;
}

//
// System configuration
//
message Config {

  optional string server_name = 1;
  repeated RelayConfig relay_config = 2;
  repeated SensorConfig sensor_config = 3;

  optional string log_file = 4;

  optional string relay_module_name = 5;
}

